antarest/study/web/study_data_blueprint.py

```diff
@@ -439,6 +444,26 @@ def remove_district(
         study_interface = study_service.get_study_interface(study)
         study_service.district_manager.remove_district(study_interface, district_id)
 
+    @bp.get(
+        "/studies/{uuid}/hydro",
+        tags=[APITag.study_data],
+        summary="Get Hydro properties for each area of the study",
+        response_model=dict[str, HydroProperties],
+        response_model_exclude_none=True,
+    )
+    def get_hydro_properties_by_area(
+        uuid: str,
+        current_user: JWTUser = Depends(auth.get_current_user),
+    ) -> dict[str, HydroProperties]:
```

Comment:
I'm not sure about returning a dict. I'd use a Pydantic model for better documentation, validation, and maintainability